serverless/architecture/serverless-event-architecture.adoc

```diff
@@ -0,0 +1,34 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-event-architecture"]
+= Knative Eventing architecture
+:context: serverless-event-architecture
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+Knative Eventing on {product-title} enables developers to use an link:https://www.redhat.com/en/topics/integration/what-is-event-driven-architecture[event-driven architecture] with serverless applications.
+An event-driven architecture is based on the concept of decoupled relationships between event producers that create events, and event  _sinks_, or consumers, that receive them.
```

Comment:
I would add here additional info that Knative Eventing is using simple HTTP POST messages to send and receive events. This is not obvious to newcomer, and a novel idea, that's great!

```suggestion
An event-driven architecture is based on the concept of decoupled relationships between event producers that create events, and event  _sinks_, or consumers, that receive them.

Knative Eventing utilizes a standard HTTP POST requests to send and receive events between event producers and consumers. That makes builing an event-driven application much simpler, as it's uses the same concepts that are required to build and test regular RESTful applications. Knative events follows link:https://cloudevents.io[CloudEvents specification], making easy to create, parse, send , and receive events in various, diffrent programming languages (Go, JavaScript, Java, Python, etc.). That gives developers additional benefit of not requireing to rely on concrete infrastructure and client libraries, and makes easy to migrate between solutions like Kafka, Camel-K etc.
```